lib/controllers/login.js

```diff
@@ -33,10 +33,24 @@ module.exports = function(req, res) {
       options = {};
     }
 
+    var hasSocialProviders = false;
+
+    // Iterate all social providers and check if there are any enabled.
+    if (config.socialProviders) {
+      for (var name in config.socialProviders) {
+        var provider = config.socialProviders[name];
+        if (provider.enabled) {
+          hasSocialProviders = true;
+          break;
+        }
+      }
+    }
+
```

Comment:
I totally disagree with you. It's just such a big difference in readability. Also underscore is apparently already used in the project and has the method [any()](http://underscorejs.org/#some) that works in exactly the same way as `lodash.any()`.